### DISCUSSION TITLE
Use separate Service Accounts for scans (enable PSPs)

### DISCUSSION THREAD

**Discussion by stone-z** [aquasecurity/starboard#787] [Development]
I was wondering if it would terribly inconvenient to provide more granular permissions for Starboard's "helper" resources. 

Currently, the same Service Account used by the operator is re-used for Pods created by Starboard's scan Jobs. In our environments, we enforce PSPs, so we have modified the Role for the Service Account to use a PSP which permits the required capabilities. 

However, since the Pods created for performing vulnerability scans and CIS benchmarks require different additional permissions (vulnerability scans require `hostPath` mounts, for example, and CIS requires many more) but are all using the same Service Account, we wind up in a situation where all of these components use a rather permissive PSP with many more privileges than actually needed by any of them.

My suggestion would be to use a separate Service Account for Pods in each type of Job. Official PSPs for them would also be helpful, but not critical.

**Comment by danielpacak** [comment]
I believe it duplicates #646

**Reply by stone-z** [reply]
Ah yep looks like it. Thanks, I missed that discussion
